frontend/wallet/src/common/Bindings.re

```diff
@@ -55,13 +55,20 @@ module Navigator = {
 };
 
 module Stream = {
-  module Readable = {
+  module Writable = {
     type t;
-    [@bs.send] external on: (t, string, Node.Buffer.t => unit) => unit = "on";
+    [@bs.send]
+    external onError: (t, [@bs.as "error"] _, string => unit) => unit = "on";
+    [@bs.send]
+    external onFinish: (t, [@bs.as "finish"] _, unit => unit) => t = "on";
   };
 
-  module Writable = {
+  module Readable = {
     type t;
+    [@bs.send] external on: (t, string, Node.Buffer.t => unit) => unit = "";
+    [@bs.send] external pipe: (t, Writable.t) => Writable.t = "";
```

Comment:
please add the name of the binding in the "" instead of leaving it an empty string